app/components/ak-button/index.scss

```diff
@@ -0,0 +1,29 @@
+.btn {
+  height: 2.5rem;
+  padding: 0.57em 0.85em;
+  border: none;
+  border-radius: var(--button-border-radius);
+  font-size: 1rem;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  justify-content: center;
+  column-gap: 0.5em;
+  line-height: 2.5rem;
+  cursor: pointer;
+
+  span {
+    //Icon size
+    font-size: 1rem;
```

Comment:
isn't this inherited from parent?

Reply:
yep, wanted to not change default properties, but makes sense to update... and done!